backend/messaging/wizard_bus.go

```diff
@@ -116,11 +116,18 @@ func (b *WizardBus) Subscribe(topic string, consumer string, sub Subscriber) (Su
 	b.topicsMu.Lock()
 	defer b.topicsMu.Unlock()
 
-	if _, ok := b.topics[topic]; !ok {
-		b.topics[topic] = b.createTopic(topic)
+	t, ok := b.topics[topic]
+	if !ok {
```

Comment:
just noticed we can refactor a bit here:
```
t, ok := b.topics[topic]
if !ok || t.IsClosed() {
    t = b.createTopic(topic)
    b.topics[topic] = t
}
```

Reply:
Oh, that cleans up nicely 👍